hera_sim/noise.py

```diff
@@ -7,23 +7,23 @@
 import aipy
 import numpy as np
 from scipy.interpolate import RectBivariateSpline
-
-this_dir, this_filename = os.path.split(__file__)
-HERA_TSKY_VS_LST_NPZ = os.path.join(this_dir, "data", "HERA_Tsky_vs_LST.npz")
-
-npz = np.load(
-    HERA_TSKY_VS_LST_NPZ
-)  # Tsky vs fq/lst from Beardsley, beam v XXX, GSM v XXX
-fqs = npz["freqs"] / 1e3
-lsts = npz["lsts"] / 12.0 * np.pi
-lsts = np.concatenate([lsts[-10:] - 2 * np.pi, lsts, lsts[:10] + 2 * np.pi])
-HERA_Tsky_xx = npz["HERA_Tsky"][0].T
-HERA_Tsky_yy = npz["HERA_Tsky"][1].T
+import aipy
+import os
+from .data import DATA_PATH
+HERA_TSKY_VS_LST_NPZ = os.path.join(DATA_PATH, 'HERA_Tsky_vs_LST.npz')
+
+npz = np.load(HERA_TSKY_VS_LST_NPZ) # Tsky vs fq/lst from Beardsley, beam v XXX, GSM v XXX
+fqs = npz['freqs'] / 1e3
+lsts = npz['lsts'] / 12. * np.pi
+lsts = np.concatenate([lsts[-10:]-2*np.pi, lsts, lsts[:10]+2*np.pi])
+HERA_Tsky_xx = npz['HERA_Tsky'][0].T
+HERA_Tsky_yy = npz['HERA_Tsky'][1].T
 HERA_Tsky_xx = np.concatenate([HERA_Tsky_xx[-10:], HERA_Tsky_xx, HERA_Tsky_xx[:10]])
 HERA_Tsky_yy = np.concatenate([HERA_Tsky_yy[-10:], HERA_Tsky_yy, HERA_Tsky_yy[:10]])
 HERA_Tsky_mdl = {}
-HERA_Tsky_mdl["xx"] = RectBivariateSpline(lsts, fqs, HERA_Tsky_xx)
-HERA_Tsky_mdl["yy"] = RectBivariateSpline(lsts, fqs, HERA_Tsky_yy)
+HERA_Tsky_mdl['xx'] = RectBivariateSpline(lsts, fqs, HERA_Tsky_xx, kx=4, ky=4)
```

Comment:
are we certain that no extrapolation is being performed? otherwise it's much better to use kx,ky=1 usually

Reply:
It may be possible to extrapolate it and get asymptotic results, but the high-order interpolation serves an important purpose, which is to suppress high fringe-rate structure in the valid LST regimes. In the case that one gets wild results from bad extrapolation it will probably be apparent. Not sure if `RectBivariateSpline` gives extrapolation warnings, but `interp1d` does, however, `RectBivariateSpline` is supposed to be faster than `interp1d`, so not sure what the tradeoff is. I'm fine w/ it as is for now..

Reply:
Sounds good. Can we add a short comment there to this effect, to warn our future selves?